src/js/api.js

```diff
@@ -0,0 +1,31 @@
+export async function getPokemons() {
+    const response = await fetch("https://pokeapi.co/api/v2/pokemon");
+    const pokemons = await response.json();
+
+    return pokemons;
```

Comment:
You can do `return await response.json()`

Reply:
Never thought about it. Good to know